src/main/less/core.less

```diff
@@ -279,3 +279,15 @@ svg:not(:root) {
         }
     }
 }
+
+.no-data-text {
+    font-family: "Helvetica Neue";
+    font-size: 24px;
+    text-anchor: middle;
+    fill: #868a8c;
+}
+
+.no-data-layout {
+    fill: white;
+    opacity: 0.6;
```

Comment:
How is this picked?

Reply:
I have taken the requirement from [here](https://wiki.cerner.com/display/UserExperience/Advanced+Graphing?preview=/1456055900/2063505555/NoData_Callouts_7.18.2019.pdf).
It is mentioned to have opacity to 0.4, I have updated to 0.6 to display better. Should I revert it back to 0.4 or 0.6 is fine?